src/style.css

```diff
@@ -1,3 +1,65 @@
+header {
+  --main-color:  #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  text-transform: uppercase;
+}
+
+.img__logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 50px;
```

Comment:
You can remove this margin and remove margin-right from the nav and add padding: 0 50px to the container instead

Reply:
still not fixed
<img width="1073" alt="Pasted Graphic" src="https://github.com/mate-academy/layout_moyo-header/assets/70220667/c293e38c-935f-46df-9869-1b46e413a62f">

Reply:
So really, I haven't fixed anything yet. Sent a pool request. After an hour I check, and here two mentors write about improvements. Amazingly. I did not send the pull request again.